ibc-apps/ics20-transfer/types/src/error.rs

```diff
@@ -14,14 +14,16 @@ use uint::FromDecStrErr;
 pub enum TokenTransferError {
     /// context error: `{0}`
     ContextError(ContextError),
-    /// invalid identifier: `{0}`
-    InvalidIdentifier(IdentifierError),
-    /// invalid trace: `{0}`
-    InvalidTrace(String),
+    /// decoding error: `{0}`
+    Decoding(DecodingError),
+    /// identifier error: `{0}`
+    Identifier(IdentifierError),
```

Comment:
The same comment I made for `ChannelError::Identifier` applies here too. 
Basically wherever we have `Decoding` variant, if the `Identifier` not used other places than `TryFrom`, we can remove it.

Reply:
[191083d](https://github.com/cosmos/ibc-rs/pull/1325/commits/191083d9aee3475afc486d2ad2cc51c9278c0ae0)